sw/device/tests/rstmgr_alert_info_test.c

```diff
@@ -456,16 +464,30 @@ static void prgm_alert_handler_round3(void) {
       kDifAlertHandlerClassA, kDifAlertHandlerClassB, kDifAlertHandlerClassC,
       kDifAlertHandlerClassD};
 
+  dif_alert_handler_class_config_t class_d_config =
+      kConfigProfiles[kDifAlertHandlerClassD];
+
+  dif_alert_handler_escalation_phase_t class_d_esc[3];
+
+  if (kDeviceType == kDeviceFpgaCw310) {
```

Comment:
@a-will i ended up doing this. It's not super elegant, let me know if something else would work better.

@alphan this is where i made a mistake, i think we DO need to distinguish between dv / asic.